.github/workflows/kyber.yml

```diff
@@ -0,0 +1,105 @@
+name: Kyber Tests
+
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  build_liboqs:
+    name: Build liboqs
+    runs-on: ubuntu-latest
+    timeout-minutes: 4
+    steps:
+      - name: Checking cache for liboqs
+        uses: actions/cache/restore@v4
+        id: cache-liboqs
+        with:
+          path: opt/liboqs/
+          key: wolfssh-liboqs-0.10.0-${{ runner.os }}
```

Comment:
For static versions of liboqs and wolfssl, I recommend setting environmental variables at the top to make it easier to update in the future. Look here for an example: https://github.com/wolfSSL/wolfssl/blob/db383519199dc9fab626df96da9ead1441bc02d7/.github/workflows/hostap-vm.yml#L15-L16. It is referenced in the action with `${{ env.LINUX_REF }}`. I would recommend something like this:
```
env:
  LIBOQS_REF: 0.10.0
  WOLFSSL_REF: v5.7.0-stable
```